azext_iot/_help.py

```diff
@@ -96,6 +96,42 @@
         az iot hub monitor-feedback -n {iothub_name} -d {device_id} -w {message_id}
 """
 
+helps[
+    "iot hub connection-string"
+] = """
+    type: group
+    short-summary: Manage IoT hub\'s connection string.
+"""
+
+helps[
+    "iot hub connection-string show"
+] = """
+    type: command
+    short-summary: Show the connection strings for an IoT hub.
+    examples:
+    - name: Show the connection string of an IoT hub using default policy and primary key.
+      text: >
+          az iot hub connection-string show -n MyIotHub
+    - name: Show all the connection string of an IoT hub using default key.
+      text: >
+          az iot hub connection-string show -n MyIotHub --all
+    - name: Show the connection string of an IoT hub included eventhub using default policy and primary key.
+      text: >
+          az iot hub connection-string show -n MyIotHub --include-eventhub
+    - name: Show the connection string of an IoT Hub using policy 'service' and secondary key.
+      text: >
+          az iot hub connection-string show -n MyIotHub --policy-name service --key-type secondary
+    - name: Show the connection strings for all IoT hubs in a resource group.
+      text: >
+          az iot hub connection-string show --resource-group MyResourceGroup
+    - name: Show the connection strings for all IoT hubs in a subscription.
+      text: >
+          az iot hub connection-string show
+    - name: Show the connection strings for an IoT hub. (autogenerated)
```

Comment:
autogenerated help stuff is not something we explicitly put in our help. There is another component responsible for creating the autogenerated help.

Reply:
fixed